docs.md

```diff
@@ -30,7 +30,97 @@ uvicorn main:app --reload
 Once the app is successfully installed, access http://127.0.0.1:8000/docs, You'll see a screen similar to the below screen shot
 <img width="1617" alt="image" src="https://github.com/cia-labs/Storage-service/assets/41864599/e8774034-5a50-4e82-9e4b-c3e84c47bdf9">
 
+---
+---
 
+# Deploying the Storage Service as a Container
 
+To deploy the storage service as a container, follow these steps:
+
+## **Create a Dockerfile:**
+
+Create a file named Dockerfile in the root directory of your project and add the following content:
+```docker
+# Use the latest version of Alpine as the base image
+FROM alpine:latest
+
+# Set the working directory inside the container
+WORKDIR /code
+
+# Copy the requirements file to the working directory
+COPY ./requirements.txt .
+
+# Copy the application files to the working directory
+COPY ./crud.py /code/
```

Comment:
Only these two required nothing else? There can be more files in the future. We should be copying the whole directory.

Reply:
yep for now

Reply:
copy the whole directory it should always be generic